include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -180,7 +180,7 @@ SECTIONS
 
 #endif /* CONFIG_CODE_DATA_RELOCATION */
 
-    SECTION_PROLOGUE(_TEXT_SECTION_NAME_2,,)
+    SECTION_PROLOGUE(_TEXT_SECTION_NAME_##_2,,)
```

Comment:
_TEXT_SECTION_NAME_ isn't a token recognized by CPP, so this doesn't work, does it?

Should be _TEXT_SECTION_NAME

Reply:
It works for GCC 6.2 for C but I'm not sure about CPP anyways changing it to _<NAME> instead does no harm. I'll change it

Reply:
Sorry, I meant the C preprocessor, not C plus plus.

Strange, I wouldn't have expected this to work.